lib/mintel/dashboards/services/omni/analytics.libsonnet

```diff
@@ -0,0 +1,162 @@
+local layout = import 'components/layout.libsonnet';
+local commonPanels = import 'components/panels/common.libsonnet';
+local promQuery = import 'components/prom_query.libsonnet';
+{
+
+  apiServiceLatency(serviceSelectorKey='service', serviceSelectorValue='$service', span=12)::
+    local config = {
+      serviceSelectorKey: serviceSelectorKey,
+      serviceSelectorValue: serviceSelectorValue,
+    };
+
+    layout.grid([
+
+    commonPanels.latencyTimeseries(
```

Comment:
should this be a panel imported here rather then being defined here in the dashboard ?

Reply:
I'm not sure yet. 

The query grouping by 'analytics_type' is pretty specific to this application right now, which is then part of the `legendFormat` too.

It's making me think whether the whole metric should be omni-specific, or whether `analytics_type` should be a bit more generic .

Reply:
I think for now, keep it here , since it's omni specific.